website/src/examples/editable-layers-advanced.js

```diff
@@ -0,0 +1,30 @@
+import React, {Component} from 'react';
```

Comment:
- Given that we will have a number of submodules, should we adopt a directory structure, `examples/editable-layers/advanced.js`?
- Can we use typescript instead of .js?

Reply:
It's a good idea. I actually started doing that and realized it required a bit more refactor of the example list component than I hoped for to make it nested. I can look into it either here or in a follow up.

Reply:
ts part resolved in #88